mix.exs

```diff
@@ -53,6 +53,7 @@ defmodule Hexpm.Mixfile do
      {:plug, "~> 1.2"},
      {:porcelain, "~> 2.0"},
      {:postgrex, ">= 0.0.0"},
+     {:qrcode, git: "https://gitlab.com/Pacodastre/qrcode"},
```

Comment:
Need to add `runtime: false` here on latest Elixir because qrcode is incorrectly saying this application has an application start callback.